io/plugins/eu.esdihumboldt.hale.io.json/src/eu/esdihumboldt/hale/io/json/JsonInstanceWriter.java

```diff
@@ -71,6 +70,14 @@ public boolean isCancelable() {
 	@Override
 	protected IOReport execute(ProgressIndicator progress, IOReporter reporter)
 			throws IOProviderConfigurationException, IOException {
+		Integer numOfDecimalPlaces = getParameter("numOfDecimalPlaces").as(Integer.class);
```

Comment:
The new parameter should also be added to the provider's definition in the extension point to document it.

Reply:
Thank you Simon for the feedback. I have updated provider's definition.